src/style.css

```diff
@@ -2,0 +1,50 @@
+body {
+  margin: 0;
+}
+
+main {
+  width: 96px;
+  height: 116px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('./images/star.svg');
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url('./images/star-active.svg');
```

Comment:
There is repetitive code setting the background-image to 'star-active.svg' for '.stars--1 .stars__star' to '.stars--5 .stars__star'. Consider using a more efficient way to achieve this, such as using a loop in a preprocessor or applying a common class to active stars.